src/ckeditorinspector.js

```diff
@@ -28,69 +24,172 @@ export default class CKEditorInspector {
 		}
 
-		CKEditorInspector._updateState();
+		return Object.keys( editors );
+	}
+
+	/**
+	 * Attaches the inspector to all CKEditor instances discovered in DOM.
+	 *
+	 * Editor instances are named `editor-1`, `editor-2`, etc..
+	 *
+	 * **Note:** This method requires CKEditor 12.3.0 or later.
+	 *
+	 * **Note:** You can pass global configuration options when attaching:
+	 *
+	 *		CKEditorInspector.attachToAll( { option: 'value', ... } );
+	 *
+	 * @param {CKEditorInspectorConfig} [options] An object of global configuration options controlling the
+	 * behavior of the inspector.
+	 * @returns {Array.<String>} Names of the editors the inspector attached to. Useful when using `CKEditorInspector.detach()`
+	 * with generated editor names.
+	 */
+	static attachToAll( options ) {
+		const domEditables = document.querySelectorAll( '.ck.ck-content.ck-editor__editable' );
+		const attachedEditorNames = [];
+
+		for ( const domEditable of domEditables ) {
+			const editor = domEditable.ckeditorInstance;
+
+			if ( editor && !CKEditorInspector._isAttachedTo( editor ) ) {
+				attachedEditorNames.push( ...CKEditorInspector.attach( editor, options ) );
+			}
+		}
 
-		return name;
+		return attachedEditorNames;
 	}
 
+	/**
+	 * Detaches the inspector from an editor instance.
+	 *
+	 *		CKEditorInspector.attach( { 'my-editor': editor } );
+	 *
+	 *		// The inspector will no longer inspect the "editor".
+	 *		CKEditorInspector.detach( 'my-editor' );
+	 *
+	 * @param {String} string Name of the editor to detach.
+	 */
 	static detach( name ) {
 		CKEditorInspector._editors.delete( name );
-		CKEditorInspector._updateState();
+		CKEditorInspector._updateEditorsState();
+	}
+
+	/**
+	 * Destroys the entire inspector application and removes it from DOM.
+	 */
+	static destroy() {
+		if ( !CKEditorInspector._wrapper ) {
+			return;
+		}
+
+		ReactDOM.unmountComponentAtNode( CKEditorInspector._wrapper );
+		CKEditorInspector._editors.clear();
+		CKEditorInspector._wrapper.remove();
+		CKEditorInspector._wrapper = null;
 	}
 
-	static _updateState() {
+	static _updateEditorsState() {
+		// Don't update state if the application was destroy()ed.
+		if ( !CKEditorInspector._isMounted ) {
+			return;
+		}
+
 		CKEditorInspector._inspectorRef.current.setState( {
 			editors: CKEditorInspector._editors
 		} );
 	}
+
+	static _mount( options ) {
+		if ( CKEditorInspector._wrapper ) {
+			return;
+		}
+
+		const container = CKEditorInspector._wrapper = document.createElement( 'div' );
+		container.className = 'ck-inspector-wrapper';
+		document.body.appendChild( container );
+
+		ReactDOM.render(
+			<InspectorUI
+				ref={CKEditorInspector._inspectorRef}
+				editors={CKEditorInspector._editors}
+				isCollapsed={options.isCollapsed}
+			/>,
+			container );
+	}
+
+	static get _isMounted() {
+		return !!CKEditorInspector._inspectorRef.current;
+	}
+
+	static _isAttachedTo( editor ) {
+		return [ ...CKEditorInspector._editors.values() ].indexOf( editor ) > -1;
```

Comment:
`includes()`?

Reply:
I'm forgetting about these new APIs :P

```suggestion
		return [ ...CKEditorInspector._editors.values() ].includes( editor );
```